contracts/test/MplRewards.t.sol

```diff
@@ -87,19 +87,19 @@ contract MplRewardsTest is DSTest {
     }
 
     function test_updatePeriodFinish() external {
-        MplRewardsOwner owner      = new MplRewardsOwner();
-        MplRewardsOwner notOwner   = new MplRewardsOwner();
+        Owner      owner           = new Owner();
+        Owner      notOwner        = new Owner();
         MplRewards rewardsContract = new MplRewards(address(0), address(1), address(owner));
 
         assertTrue(!notOwner.try_mplRewards_updatePeriodFinish(address(rewardsContract), block.timestamp + 30 days));
         assertTrue(    owner.try_mplRewards_updatePeriodFinish(address(rewardsContract), block.timestamp + 30 days));
     }
 
     function test_recoverERC20() external {
-        MplRewardsOwner owner      = new MplRewardsOwner();
-        MplRewardsOwner notOwner   = new MplRewardsOwner();
-        SomeERC2258 someToken      = new SomeERC2258("ST", "ST");
-        MplRewards rewardsContract = new MplRewards(address(0), address(1), address(owner));
+        Owner       owner           = new Owner();
+        Owner       notOwner        = new Owner();
+        SomeERC2258 someToken       = new SomeERC2258("ST", "ST");
+        MplRewards  rewardsContract = new MplRewards(address(0), address(1), address(owner));
```

Comment:
Alphabetical

Reply:
Alphabetical for all of these, based on type and then variable name

Reply:
Ditto.